source/PlayerInfo.cpp

```diff
@@ -3748,6 +3788,10 @@ void PlayerInfo::Save(const string &path) const
 			out.Write(plugin.first);
 		out.EndChild();
 	}
+
+	const auto stop = chrono::high_resolution_clock::now();
+	fprintf(stderr, "Saved in %lld us\n",
```

Comment:
is this part of your debuging?

Reply:
Yes indeed; I wanted to make sure the often unnecessary serialization didn't have a noticable performance impact.

Reply:
you might want to remove it though

Reply:
Certainly